--- build/tomcat-hosted-ntropa-builder/apache-tomcat-5.5.20/bin/catalina.sh	2006-09-12 16:10:27.000000000 +0100
+++ catalina-edited/catalina.sh	2008-02-09 22:40:03.000000000 +0000
@@ -72,7 +72,7 @@
 [ -z "$CATALINA_HOME" ] && CATALINA_HOME=`cd "$PRGDIR/.." ; pwd`
 
 if [ -r "$CATALINA_HOME"/bin/setenv.sh ]; then
-  . "$CATALINA_HOME"/bin/setenv.sh
+  . "$CATALINA_HOME"/bin/setenv.sh "$1"
 fi
 
 # For Cygwin, ensure paths are in UNIX format before anything is touched
